server/server.js

```diff
@@ -157,8 +157,16 @@ app.use(session({
 var exonFile = dataPath + 'exons.bin';
 
 /**
+ * @type {!Array<string>}
+ */
+var allowedOrigins = [
+  'https://localhost',
+  'file://'
+];
+
+/**
  * Sends back JSON response.
- * @param {Object|user.Error|undefined} data Server response data.
+ * @param {Object|user.Error|string|undefined} data Server response data.
```

Comment:
Actually it is a little bit weird that this needs user.Error. If it needs user.Error, then it could need binding.Error/network.Error/... as well. I will leave it for your discretion.